practice-exercises/01-basic-challenges-1/01-hello-world/hello-world.ts

```diff
@@ -0,0 +1,5 @@
+export function helloWorld(str: string): string {
+  return str
+}
+
+helloWorld('Hello World')
```

Comment:
You are supposed to implement the function `helloWorld` so that it returns the string 'Hello World'. Don't call it. The test will call it.

From what I see, as this code is written the test should fail.